src/shared/config/validation_pipe.config.ts

```diff
@@ -0,0 +1,10 @@
+import { ValidationPipeOptions } from '@nestjs/common';
+
+export const validationPipeOptions: ValidationPipeOptions = {
+  whitelist: true,
+  forbidNonWhitelisted: true,
+  transform: true,
+  transformOptions: {
+    enableImplicitConversion: true
```

Comment:
I'm wondering, why did you enable implicit conversion?

Reply:
When sending take and skip it was not being converted to number. Is there a better solution for this?